docs/cloud/managing-airbyte-cloud.md

```diff
@@ -85,22 +85,18 @@ To switch between workspaces:
 3. Click the name of the workspace you want to switch to.
 
 ### Choose your default data residency
-Default data residency allows you to choose where your data is processed. When you set the default data residency, it is applied to all new connections, but it does not affect existing connections. 
+Default data residency allows you to choose where your data is processed. When you set the default data residency, it is applied to all new connections, but it does not affect existing connections or any currently running syncs. 
 
 For individual connections, you can choose a data residency that is different from the default. You can do this in the [connection settings](#choose-the-data-residency-for-a-connection) or when you create a [new connection](https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#set-up-a-connection).
 
-:::note 
-
-Your data is processed in the chosen residency, but some data is still stored in the US.
-
-:::
+Your data is processed on a data plane in the chosen residency, but some data, like data associated with the sync mode, cursor, and primary key, is still stored in the US on our control plane. For streams with data that must stay in the chosen data residency, do not configure sync modes that use cursor or primary key.
```

Comment:
```suggestion
Although your data is processed on a data plane in the chosen residency, configuration data e.g. data associated with the sync mode, cursor, and primary key, is still stored on our control plane in the US. Data that must stay in the chosen data residency should not be used as a stream's cursor or primary key.
```

Reply:
Some reword to clarify that the streams can use cursors/primary keys as long as the sensitive data is neither.